tests/reporter/kubernetes.go

```diff
@@ -134,6 +142,10 @@ func (r *KubernetesReporter) Dump(duration time.Duration) {
 	r.logDomainXMLs(virtCli, vmis)
 
 	r.logSRIOVInfo(virtCli)
+
+	r.logNodeCommands(virtCli)
```

Comment:
Is there anything else other than network info that is retrieved from `logNodeCommands`, `logVirtLauncherCommands`, and `logVMICommands` ?
Perhaps it would be better to put it in a "network context" and call those functions :
`logNodeNetworkInfo`, `logVirtLauncherNetworkInfo`, and `logVMINetworkInfo`?
or, the same but with 'NetworkState` instead of `Commands`?

Reply:
Answering on all the comments at once:
its almost true that they run only network commands but
1. dmesg is not a network command, so maybe not perfectly true.
2. we might want to add them to the future commands which are not network

having said that, it dumps the info in network folder

considered that, wdyt we should do ? (about all comments, since there are two groups ,this one, and the inner functions of the other comments)

Reply:
I see your point. Fine by me to leave it as is